src/components/Icon/Icon.d.ts

```diff
@@ -0,0 +1,41 @@
+import IconMap from './Icon.map';
+
+export type IconType = keyof typeof IconMap;
+
+export interface StyledContainerProps {
```

Comment:
no need to export as the file type is `.d.ts` so, any types mentioned here would automatically be available in all `ts` files without explicit export/import

Reply:
okay, didn't know about this issue with storyBook